sdk/documentintelligence/azure-ai-documentintelligence/samples/aio/sample_analyze_addon_barcodes_async.py

```diff
@@ -71,17 +71,18 @@ async def analyze_barcodes():
                 features=[DocumentAnalysisFeature.BARCODES],
                 content_type="application/octet-stream",
             )
-        result = await poller.result()
+        result: AnalyzeResult = await poller.result()
```

Comment:
result should already be inferred as an `AnalyzeResult`. Did you add this for clarity?

Reply:
Yes, no errors in mypy for it.